### PR TITLE
TGA: get logits out, also fix gpt2

### PR DESCRIPTION
**Pull request by jaseweston**

### CODE REVIEW

File: parlai/agents/hugging_face/dict.py [hunk 1]
```diff
@@ -204,6 +204,9 @@ def override_special_tokens(self, opt):
         self.ind2tok[self.start_idx] = self.start_token
         self.ind2tok[self.null_idx] = self.null_token
 
+        self.unk_token = self.hf_tokenizer.unk_token
```
Comment (jxmsML) [inline parlai/agents/hugging_face/dict.py:207]: Kurt has this fix in an earlier pull request https://github.com/facebookresearch/ParlAI/pull/5016/files

**Review by klshuster — APPROVED** [review]
